include/RAJA/policy/sequential/scan.hpp

```diff
@@ -143,6 +149,8 @@ concepts::enable_if<type_traits::is_sequential_policy<ExecPolicy>> exclusive(
   decltype(*begin) agg = v;
   OutIter o = out;
   *o++ = v;
+
+  RAJA_NoSIMD
```

Comment:
Should I have no-vect pragma's for sequential scan? Or would this be unnecessary?

Reply:
I don't think it will hurt anything. Include them for consistency.